android/anc/src/test/java/org/smartregister/fhircore/anc/ui/family/details/FamilyDetailScreenTest.kt

```diff
@@ -135,75 +142,41 @@ class FamilyDetailScreenTest : RobolectricTest() {
 
   @Test
   fun testEncounterHeaderComponent() {
-    composeRule.setContent { EncounterHeader { listenerObjectSpy.onSeeAllEncounterClick() } }
     // Encounter heading is displayed
     composeRule.onNodeWithText("Encounters".uppercase()).assertExists()
-    composeRule.onNodeWithText("Encounters".uppercase()).assertIsDisplayed()
 
     // See All encounters buttons is displayed
-    composeRule.onNodeWithText("See All".uppercase()).assertExists()
-    composeRule.onNodeWithText("See All".uppercase()).assertIsDisplayed()
-
-    // clicking see all button should call 'onSeeAllEncounterClick' method of 'listenerObjectSpy'
-    composeRule.onNodeWithText("See All".uppercase()).performClick()
-    verify { listenerObjectSpy.onSeeAllEncounterClick() }
```

Comment:
Any reason why the click listener assertions were removed?

Reply:
previously having some issues with a click it was not working because i moved `setContent` to the `setUp` method. Now all changes have been reverted and it's working fine